spec/rubocop/cop/rspec/metadata_style_spec.rb

```diff
@@ -0,0 +1,234 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::MetadataStyle do
+  context 'with `EnforcedStyle: symbol`' do
+    let(:cop_config) do
+      { 'EnforcedStyle' => 'symbol' }
+    end
+
+    context 'with symbol metadata' do
+      it 'registers no offense' do
+        expect_no_offenses(<<~RUBY)
+          describe 'Something', :a do
+          end
+        RUBY
+      end
+    end
+
+    context 'with hash metadata' do
+      it 'registers offense' do
+        expect_offense(<<~RUBY)
+          describe 'Something', a: true do
+                                ^^^^^^^ Use symbol style for metadata.
+          end
+        RUBY
+
+        expect_correction(<<~RUBY)
+          describe 'Something', :a do
+          end
+        RUBY
+      end
+    end
+
+    context 'with hash metadata with preceding symbol metadata' do
+      it 'registers offense' do
+        expect_offense(<<~RUBY)
+          describe 'Something', :b, a: true do
```

Comment:
aren't we missing spec that mixes both preceding symbol metadata, and having other hash metadata?
What about having 2 hash keys with values true?